### ISSUE TITLE
[ZF2.4.13] Hostname-Validator not allow .gmbh-TLD

### ISSUE THREAD

**Issue by ssh-pw** [zendframework/zendframework#7772]
Hi,

the current ZF2.4.13-release not support the TLD ".gmbh", although the domain is listed on "http://www.iana.org/domains/root/db/".

Temporary solution is to write a wrapper-class. Can this be fixed in the 2.4.14-release?

Thanks and many regards,
patric

**Comment by Ocramius** [comment]
Don't think this counts as bugfix here, as we're tracking a moving target
(TLDs) in newer minor releases.

Therefore you'll probably won't get it in 2.4, but rather in the last
versions of the hostname validator.

I think the issue is broader though, and we should somehow automate the
feeding of TLDs into the validator, instead of having them tracked by VCS.

Marco Pivetta

http://twitter.com/Ocramius

http://ocramius.github.com/

**Comment by froschdesign** [comment]
> I think the issue is broader though, and we should somehow automate the
feeding of TLDs into the validator, instead of having them tracked by VCS.

See: https://github.com/zendframework/zend-validator/blob/master/bin/update_hostname_validator.php

**Comment by Ocramius** [comment]
Yes, the idea is to move it to a post-install hook, instead of having it in
VCS

Marco Pivetta

http://twitter.com/Ocramius

http://ocramius.github.com/

**Comment by froschdesign** [comment]
@Ocramius 
> Yes, the idea is to move it to a post-install hook, instead of having it in
VCS

Good idea, but wrong issue tracker for this! 😉
